Sources/App/Providers/BinarySizeProvider/AppManager.swift

```diff
@@ -53,6 +58,15 @@ final class AppManager {
 
     func cloneEmptyApp() throws {
         do {
+            if fileManager.fileExists(atPath: emptyAppDirectoryPath) {
+                if verbose {
+                    console.lineBreakAndWrite(
+                        "Removing existing EmptyApp directory: \(emptyAppDirectoryPath)"
+                    )
+                }
+                try fileManager.removeItem(atPath: emptyAppDirectoryPath)
+            }
```

Comment:
Nice!!
I was wondering if wouldn't be a good idea to have the directory cleanup as a separate function and step.
This would allow to call it from `SizeMeasurer` as a separate step under `measureEmptyAppSize` and show it when verbose mode if off as part of the loading steps .. `[1/7] - doing something`.

What do you think?

Reply:
Sure! Code updated.